Example/src/screens/StackPresentation.tsx

```diff
@@ -0,0 +1,163 @@
+      <Button title="modal" onPress={() => navigation.navigate('Modal')} />
+      <Button
+        title="transparentModal"
+        onPress={() => navigation.navigate('TransparentModal')}
+      />
+      <Button
+        title="containedModal"
+        onPress={() => navigation.navigate('ContainedModal')}
+      />
+      <Button
+        title="containedTransparentModal"
+        onPress={() => navigation.navigate('ContainedTransparentModal')}
+      />
+      <Button
+        title="fullScreenModal"
+        onPress={() => navigation.navigate('FullScreenModal')}
+      />
+      <Button
+        title="formSheet"
+        onPress={() => navigation.navigate('FormSheet')}
+      />
+      <Button onPress={() => navigation.pop()} title="🔙 Back to Examples" />
+    </ScrollView>
+  );
+};
+
+interface FormScreenProps {
+  navigation: NativeStackNavigationProp<ParamListBase>;
+}
+
+const FormScreen = ({navigation}: FormScreenProps): JSX.Element => (
+  <View style={{...styles.container, backgroundColor: 'white'}}>
+    <Form />
+    <Button title="Go back" onPress={() => navigation.goBack()} />
+  </View>
+);
+
+interface ModalScreenProps {
+  navigation: NativeStackNavigationProp<ParamListBase>;
+}
+
+const ModalScreen = ({navigation}: ModalScreenProps): JSX.Element => (
+  <View style={styles.container}>
+    <Choose />
+    <Button title="Go back" onPress={() => navigation.goBack()} />
+  </View>
+);
+
+interface FullScreenModalProps {
+  navigation: NativeStackNavigationProp<ParamListBase>;
+}
+
+const FullScreenModalScreen = ({
+  navigation,
+}: FullScreenModalProps): JSX.Element => (
+  <View style={{flex: 1}}>
+    <ImageBackground
+      style={styles.image}
+      source={require('../assets/trees.jpg')}>
+      <Button title="Go back" onPress={() => navigation.goBack()} />
+    </ImageBackground>
+  </View>
+);
+
+const Stack = createNativeStackNavigator<StackParamList>();
+
+const App = (): JSX.Element => (
+  <Stack.Navigator
+    screenOptions={{
+      statusBarStyle: 'dark',
+      headerHideBackButton: true,
+      direction: I18nManager.isRTL ? 'rtl' : 'ltr',
+    }}>
+    <Stack.Screen
+      name="Main"
+      component={MainScreen}
+      options={{title: 'Stack Presentation'}}
+    />
+    <Stack.Screen
+      name="Push"
+      component={FormScreen}
+      options={{stackPresentation: 'push'}}
+    />
+    <Stack.Screen
+      name="Modal"
+      component={ModalScreen}
+      options={{stackPresentation: 'modal'}}
+    />
+    <Stack.Screen
+      name="TransparentModal"
+      component={Alert}
+      options={{stackPresentation: 'transparentModal', headerShown: false}}
+    />
+    <Stack.Screen
+      name="ContainedModal"
+      component={ModalScreen}
+      options={{stackPresentation: 'containedModal'}}
+    />
+    <Stack.Screen
+      name="ContainedTransparentModal"
+      component={Dialog}
+      options={{stackPresentation: 'containedTransparentModal'}}
```

Comment:
I think it also needs `headerShown: false`

Reply:
Fixed in https://github.com/software-mansion/react-native-screens/pull/827/commits/4cca989fd6ae99ccdf24266248d242caf1e64423